ext/unicorn_http/extconf.rb

```
@@ -2,6 +2,21 @@
+def generate_ragel_source
+  src = File.expand_path('unicorn_http.c', __dir__)
+  return if File.exist?(src)
+
+  ragel_cfg = with_config('ragel')
+  ragel_cfg = ragel_cfg.strip if ragel_cfg.respond_to?(:strip)
+  ragel = if ragel_cfg.is_a?(String) && !ragel_cfg.empty? && ragel_cfg !~ /\A(?:yes|no)\z/i
+    ragel_cfg
+  end
+  ragel ||= find_executable('ragel')
+  end
+end
```

RuboCop (1.80.2)

[fatal] unexpected token kEND
(Using Ruby 2.5 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)